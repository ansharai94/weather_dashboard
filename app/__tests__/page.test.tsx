import { render, screen } from "@testing-library/react";
import Home from "../page";

vi.mock("@/components/weather-dashboard", () => ({
  default: function MockWeatherDashboard() {
    return (
      <div data-testid="weather-dashboard">Weather Dashboard Component</div>
    );
  },
}));
describe("Home Page", () => {
  it("matches snapshot", () => {
    render(<Home />);
    expect(screen.getByTestId("weather-dashboard")).toBeInTheDocument();
  });
  it("matches snapshot", () => {
    const { container } = render(<Home />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
