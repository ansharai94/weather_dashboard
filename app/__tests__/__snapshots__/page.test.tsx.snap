// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Home Page > matches snapshot 1`] = `
<div
  data-testid="weather-dashboard"
>
  Weather Dashboard Component
</div>
`;
